### ISSUE TITLE
about class-free guidance

### ISSUE THREAD

**Issue by renrenzsbbb** [hpcaitech/Open-Sora#843]
thanks for your great work and opening project. I find that in sampling.py, line 194 which is img = torch.cat([cond_x, cond_x, cond_x], dim=0); so that for cond, uncond, uncond_2 = pred.chunk(3, dim=0), it's conditioned on img which is not consitent with uncond_2 meanings. can you give me you advice?

**Comment by SimonWXW** [comment]
you could check the code line 187 - 189: 
```python
cond = torch.cat((masks, masked_ref), dim=1)
cond = pack(cond, patch_size=patch_size)
kwargs["cond"] = torch.cat([cond, cond, torch.zeros_like(cond)], dim=0)
```
uncond_2 corresponds to the part: torch.zeros_like(cond)
